coral-component-card/src/scripts/CardProperty.js

```diff
@@ -73,6 +73,12 @@ const CardProperty = Decorator(class extends BaseComponent(HTMLElement) {
 
     // removes the icon element from the DOM since there is no valid icon. this causes the content to have the
     // correct styling
+    
+    if(this._elements.icon.icon === 'globeRemove') {
+      this._elements.icon.removeAttribute('aria-label');
```

Comment:
Please avoid check related to single icon. The code should be generic and not specified to single icom

Reply:
@Pareesh  the ticket required to remove the aria label from the specific icon, will check with manager, thank you for the comment